### DISCUSSION TITLE
When will translations apply at release version?

### DISCUSSION THREAD

**Discussion by jujumilk3** [fastapi/fastapi#5180] [Questions] [answered]
I translated some docs.

But it doesn't applying at release versions for a long time.
It make me feel so meaningless... 😰

**Comment by odiseo0** [comment]
It's ok, don't worry, some things takes longer to see the light

**Reply by jujumilk3** [reply]
🥲

**Comment by tiangolo** [comment, marked as answer]
I have to wait for other native speakers to review the PRs, ask for changes or approve them. Once there are two native speakers approving PR, then I can merge it! :tada: 

You can read more here: https://fastapi.tiangolo.com/contributing/#translations
